modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/model/listener/test/UserModelListenerTest.java

```diff
@@ -154,6 +202,54 @@ private void _assertEqualsSorted(
 	private ObjectDefinitionLocalService _objectDefinitionLocalService;
 
 	private ServiceRegistration<SystemObjectDefinitionManager>
-		_serviceRegistration;
+		_systemObjectDefinitionManagerServiceRegistration;
+
+	@Inject
+	private UserLocalService _userLocalService;
+
+	private class TestAuditMessageProcessor implements AuditMessageProcessor {
+
+		@Override
+		public void process(AuditMessage auditMessage) {
+			Assert.assertNotNull(auditMessage);
+
+			JSONObject additionalInfoJSONObject =
+				auditMessage.getAdditionalInfo();
+
+			String authType = String.valueOf(
+				additionalInfoJSONObject.get("authType"));
+
+			Assert.assertEquals("emailAddress", authType);
+
+			String reason = String.valueOf(
+				additionalInfoJSONObject.get("reason"));
+
+			Assert.assertEquals("User does not exist", reason);
+
+			Assert.assertEquals(
+				EventTypes.LOGIN_DNE, auditMessage.getEventType());
+		}
+
+	}
+
+	private class UserLocalServiceTestBundleActivator
+		implements BundleActivator {
+
+		@Override
+		public void start(BundleContext bundleContext) {
+			_auditMessageProcessorServiceRegistration =
+				_bundleContext.registerService(
+					AuditMessageProcessor.class, _auditMessageProcessor,
+					HashMapDictionaryBuilder.<String, Object>put(
+						"eventTypes", EventTypes.LOGIN_DNE
+					).build());
+		}
+
+		@Override
+		public void stop(BundleContext bundleContext) {
+			_systemObjectDefinitionManagerServiceRegistration.unregister();
```

Comment:
@thallescezar is `_systemObjectDefinitionManagerServiceRegistration` supposed to be unregistered here?

Reply:
This was a mistake. The right on should be _auditMessageProcessorServiceRegistration. Going to fix it :)